src/events/guildMemberRemove.ts

```diff
@@ -13,16 +13,28 @@ export default async function guildMemberRemoveHandler(member: GuildMember | Par
 	const user = member.user;
 	
 	const auditLogs = await guild.fetchAuditLogs({
-		limit: 1
+		limit: 10
 	});
 
+	const auditLog = auditLogs.entries.find(log => {
+		return (log.action === AuditLogEvent.MemberBanAdd || log.action === AuditLogEvent.MemberKick) &&
+		log.targetId === member.id &&
+		Date.now() - log.createdTimestamp < 2000;
+	});
```

Comment:
Not an "issue" but just from a stylistic standpoint I try to avoid stuff like this (both the very long `if` style checks with many conditions, and directly returning on them). It makes it a bit harder to parse at a glance imo

This can be refactored to use a fix of `filter` to narrow the log actions, and then `find` to get the correct one

Reply:
The filter is still going to be long, do you want to break the `&&`d parts up into separate filters, or would you still `&&` those together on the `filter`?
```ts
auditLogs.entries
    .filter(log => Date.now() - log.createdTimestamp < 2000)
    .filter(log => log.action === AuditLogEvent.MemberBanAdd || log.action === AuditLogEvent.MemberKick)
    .find(log => log.targetId === member.id);
```

Reply:
Using what you suggested seems fine, using two `filter`s and a `find` breaks it up into more readable components which clearly describe the checks at a glance